ui/focusable.js

```diff
@@ -42,9 +42,18 @@ $.ui.focusable = function( element, hasTabindex ) {
 		"a" === nodeName ?
 			element.href || hasTabindex :
 			hasTabindex ) &&
-		$( element ).is( ":visible" ) && $( element ).css( "visibility" ) === "visible";
+		$( element ).is( ":visible" ) && visible( $( element ) );
 };
 
+function visible( element ) {
+	var visibility = element.css( "visibility" );
+	while ( visibility === "inherit" ) {
```

Comment:
There is no possible infinite loop here. If the element doesn't exist, it can't have a `visibility` of `"inherit"`.